<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Assignments</class>
    <widget class="QMainWindow" name="Assignments">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Assignments</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QTableWidget" name="table_Assignments"/>
                </item>
                <item>
                    <widget class="QTextBrowser" name="text_AssignmentNotes">
                        <property name="maximumSize">
                            <size>
                                <width>16777215</width>
                                <height>120</height>
                            </size>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                        <item>
                            <widget class="QPushButton" name="btn_Start">
                                <property name="text">
                                    <string>&amp;Start</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer_4">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="btn_Done">
                                <property name="text">
                                    <string>&amp;Done</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer_3">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="btn_Abort">
                                <property name="text">
                                    <string>&amp;Abort</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer_2">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="btn_Delete">
                                <property name="text">
                                    <string>Delete</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <spacer name="horizontalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Orientation::Horizontal</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>40</width>
                                        <height>20</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <widget class="QPushButton" name="btn_Close">
                                <property name="text">
                                    <string>&amp;Close</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>btn_Close</sender>
            <signal>clicked()</signal>
            <receiver>Assignments</receiver>
            <slot>close()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>699</x>
                    <y>551</y>
                </hint>
                <hint type="destinationlabel">
                    <x>399</x>
                    <y>299</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>
